public/js/data.js

```diff
@@ -88,18 +52,15 @@ $(document).ready( function () {
                 }
             };
     
-            var levelChart = new google.visualization.PieChart(document.getElementById('levelChart'));
+            const levelChart = new google.visualization.PieChart(document.getElementById('levelChart'));
```

Comment:
jQuery tip: use [ID selectors](https://api.jquery.com/id-selector/) instead of `document.getElementById`